src/site/content/en/blog/launch-handler/index.md

```diff
@@ -0,0 +1,244 @@
+
+Both `route_to` and `navigate_existing_client` also accept a list (array) of values, where the first
+valid value will be used. This is to allow new values to be added to the spec without breaking
+backwards compatibility with existing implementations.
+
+For example, if the value `"matching-url-client"` were added, sites would specify
+`"route_to": ["matching-url-client", "existing-client"]` to continue to control the behavior of
+older browsers that did not support `"matching-url-client"`.
+
+### The `window.launchQueue` interface
+
+If the app has declared that it wants to handle launches in an existing client (by specifying
+`"route_to": "existing-client"`), it can imperatively do something with incoming launch URLs.
+This is where the `launchQueue` comes into play. To access launch target URLs, a site needs to
+specify a consumer for the `window.launchQueue` object, which is then passed the target URL via the
+`launchParams.targetURL` field. Launches are queued until they are handled by the specified
+consumer, which is invoked exactly once for each launch. In this manner, every launch is handled,
+regardless of when the consumer was specified. The code snippet below shows a fictive audio player
+PWA that extract a song ID from a target URL that it is potentially passed upon launch.
+
+```js
+launchQueue.setConsumer((launchParams) => {
+  const songID = extractSongId(launchParams.targetURL);
+  if (songID) {
+    playSong(songID);
+  }
+});
+```
+
+## Demo
+
+You can see a demo of the Launch Handler API in action in the [PWA Launch Handler Demo][demo]. Be
+sure to check out the [source code][demo-source] of the application to see how it uses the Launch
+Handler API.
+
+<!--lint disable no-literal-urls -->
+1. Install the app on a Chrome OS device.
+1. Send yourself a link in a chat application of the form
+   <code>https://launch-handler.glitch.me?track=<strong>https://example.com/music.mp3</strong></code>.
+   (You can customize
+   <code><strong>https://example.com/music.mp3</strong></code> for any URL pointing to an audio file, for example,
+   <a href="https://launch-handler.glitch.me?track=https://cdn.glitch.me/3e952c9c-4d6d-4de4-9873-23cf976b422e%2Ffile_example_MP3_700KB.mp3?v=1638795977190"><code>https://launch-handler.glitch.me?track=https://cdn.glitch.me/3e952c9c-4d6d-4de4-9873-23cf976b422e%2Ffile_example_MP3_700KB.mp3?v=1638795977190</code></a>).
+1. Click the link in your chat app and notice how _Musicr 2.0_ opens and plays the track.
+1. Click the link in your chat app again and notice how always just one instance of Musicr 2.0 exists.
+<!--lint enable no-literal-urls -->
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/nFgE69N5VTYBDon1DGmB.png", alt="Musicr 2.0 web application launched and playing music it handled from the launch params.", width="800", height="235" %}
+
+## Security and permissions
+
+The Chromium team has designed and implemented the Launch Handler API using the core principles
```

Comment:
```suggestion
The Chromium team designed and implemented the Launch Handler API using the core principles
```

Reply:
I should have caught this a long time ago. I've [opened a PR](https://github.com/GoogleChrome/web.dev/pull/7033) to fix this elsewhere.